utils/gtrnadb.py

```diff
@@ -176,7 +223,8 @@ def get_trnascan_cm(domain, isotype):
 
     result = runner.run(f"cmfetch -o {cm_output} {cm_library} {cm_name}")
     if result:
-        os.remove(cm_output)
+        if cm_output.exists():
+            cm_output.unlink()
```

Comment:
no need for the `if` statement _if_ you use `cm_output.unlink(missing_ok=True)`

Reply:
You are right, thanks!